core/src/components/header/header.ionic.scss

```diff
@@ -0,0 +1,13 @@
+@use "./header.common";
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+
+// Ionic Header
+// --------------------------------------------------
+
+ion-header {
+  z-index: 10;
+
+  .header-ionic {
+    box-shadow: #{globals.$ionic-elevation-200};
+  }
```

Comment:
This needs to be moved outside of `ion-header` for it to work. By placing it inside, it targets `ion-header > .header-ionic`.

Reply:
good catch, thanks 👍